skbio/stats/tests/test_composition.py

```diff
@@ -0,0 +1,117 @@
+from __future__ import absolute_import, division, print_function
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from unittest import TestCase, main
+
+import numpy as np
+from numpy.testing import assert_array_almost_equal
+from skbio.stats.composition import (_closure, multiplicative_replacement,
+                                     perturb, perturb_inv, power,
+                                     clr, centralize)
+
+
+class CompositionTests(TestCase):
+
+    def setUp(self):
+        self.data1 = np.vstack((
+            np.array([2, 2, 6]),
+            np.array([4, 4, 2])))
+        self.data2 = np.array([2, 2, 6])
+
+        self.data3 = np.vstack((
+            np.array([1, 2, 3, 0, 5]),
+            np.array([1, 0, 0, 4, 5]),
+            np.array(range(1, 6))))
+        self.data4 = np.array([1, 2, 3, 0, 5])
+
+    def test_closure(self):
+        assert_array_almost_equal(_closure(self.data1),
```

Comment:
Use `npt.assert_allclose` instead (there are several instances of this).

From the docstring of the function you're using:
```
.. note:: It is recommended to use one of `assert_allclose`,
          `assert_array_almost_equal_nulp` or `assert_array_max_ulp`
          instead of this function for more consistent floating point
          comparisons.
```